tests/pipeline/test_pipeline.py

```diff
@@ -2161,3 +2161,27 @@ def test_yielding_empty_list_creates_table() -> None:
             rows = list(cur.fetchall())
             assert len(rows) == 1
             assert rows[0] == (1, None)
+
+
+def test_staging_dataset_truncate() -> None:
+    @dlt.resource(write_disposition="merge", merge_key="id")
```

Comment:
cool! but remember to enable deletion in config

Reply:
Done